elements/pl-card/pl-card.py

```diff
@@ -0,0 +1,71 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+
+HEADER_DEFAULT = ""
+TITLE_DEFAULT = ""
+SUBTITLE_DEFAULT = ""
+FOOTER_DEFAULT = ""
+IMG_TOP_SRC_DEFAULT = ""
+IMG_TOP_ALT_DEFAULT = ""
+IMG_BOTTOM_SRC_DEFAULT = ""
+IMG_BOTTOM_ALT_DEFAULT = ""
+WIDTH_DEFAULT = "auto"
+
+
+def prepare(element_html: str, data: pl.QuestionData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    required_attribs = []
+    optional_attribs = [
+        "header",
+        "title",
+        "subtitle",
+        "footer",
+        "img-top-src",
+        "img-top-alt",
+        "img-bottom-src",
+        "img-bottom-alt",
+        "width",
+    ]
+    pl.check_attribs(element, required_attribs, optional_attribs)
+
+
+def render(element_html: str, data: pl.QuestionData) -> str:
```

Comment:
Almost missed this: for type checking to be enabled you need to add the directory of this element to `pyproject.toml`

Reply:
Done 72f796d